test/utils/utils.ts

```diff
@@ -0,0 +1,728 @@
+import * as utils from "@across-protocol/contracts-v2/dist/test-utils";
+
+import { TokenRolesEnum } from "@uma/common";
```

Comment:
I've proposed to define this locally in this PR in relayer-v2: https://github.com/across-protocol/relayer-v2/pull/956/files

The rationale for defining it locally is that it's isolated to `test/utils/utils.ts`.

wdyt on incorporating that here, or alternatively handling it in a follow-up commit ?

Reply:
Yeah I think that would be fine - it may be beneficial to propose this change as a PR against this one so that we can see the diff.